src/backend/db/users.js

```diff
@@ -30,7 +30,7 @@ export const users = [
     firstName: "John",
     lastName: "Doe",
     email: "johndoe@gmail.com",
-    password: "12345678",
+    password: 12345678,
```

Comment:
Password should be string, right?

Reply:
Hey Mahesh, using string gave me a 404 error. It started working fine when I changed it to be a number.